### PR TITLE
ts/env/docker/eslint migration

### PR DESCRIPTION
**Pull request by Niputi**
- [✔] typescript
- [✔] env file
- [✔] docker

### CODE REVIEW

**Comment by velddev** [comment]
Make sure to remove `dist/` for this commit

File: .env.schema [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# api
+API_PORT=
+
+# postgress
```
Comment (velddev) [inline .env.schema]: postgres*

File: .env.schema [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+DATABASE_PORT=
+DATABAE_PASSWORD=
+DATABAE_USER=
+DATABAE_DATABAE=
```
Comment (velddev) [inline .env.schema]: DATABASE :')

File: src/routes/users.router.ts [hunk 3]
```diff
@@ -26,7 +25,7 @@ Canvas.registerFont("./assets/fonts/YuGothL.ttc", {
 });
 
 // Calculates Miki level from experience
-function CalculateLevel(exp)
+function CalculateLevel(exp : number)
```
Comment (velddev) [inline src/routes/users.router.ts]: Make sure to keep consistent in style

File: src/routes/users.router.ts [hunk 4]
```diff
@@ -52,16 +51,18 @@ function CalculateExp(level)
...
+Router().get('/api/custom', async (req, res) => {
+    var xml = fs.readFileSync("../../test.xml", "utf8");
     console.log(xml);
     parseString(xml, function (err, result) {
         if(err) console.log(err);
 
         var avatarUrl = "https://cdn.miki.ai/avatars/121919449996460033.png"
 
+        //@ts-ignore
         loadPNG(avatarUrl, (avatar) =>
```
Comment (velddev) [inline src/routes/users.router.ts]: maybe promisify loadPNG?

**Review by velddev — COMMENTED** [review]
Check the changes here

File: src/routes/users.router.ts (hunk 3 above)
Comment (Niputi) [reply src/routes/users.router.ts]: :thinking: it's the original style

File: .env.schema [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# api
+API_PORT=
+
+# postgres
+DATABASE_PORT=
+DATABAE_PASSWORD=
+DATABAE_USER=
```
Comment (velddev) [inline .env.schema]: Spelling issues

File: .env.schema [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# api
+API_PORT=
+
+# postgres
+DATABASE_PORT=
+DATABAE_PASSWORD=
```
Comment (velddev) [inline .env.schema]: Spelling issues

File: src/index.d.ts [hunk 7]
```diff
@@ -0,0 +1,9 @@
+declare namespace NodeJS {
+    export interface ProcessEnv {
+        API_PORT: string,
+        DATABASE_PORT: string,
+        DATABAE_PASSWORD: string,
```
Comment (velddev) [inline src/index.d.ts]: Spelling issues

File: src/index.d.ts [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+        DATABAE_USER: string,
```
Comment (velddev) [inline src/index.d.ts]: Spelling issues

File: src/index.d.ts [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+        DATABAE: string
```
Comment (velddev) [inline src/index.d.ts]: Spelling issues

File: src/routes/users.router.ts [hunk 10]
```diff
@@ -1,18 +1,17 @@
...
 const pool = new Pool({
-    user: global.config.user,
-    host: global.config.host,
-    database: global.config.database,
-    password: global.config.password,
-    port: global.config.port,
+    user: process.env.DATABAE_USER,
+    host: process.env.DATABAE_HOST,
+    database: process.env.DATABAE,
+    password: process.env.DATABAE_PASSWORD,
```
Comment (velddev) [inline src/routes/users.router.ts]: Spelling issues

File: src/routes/users.router.ts [hunk 11]
```diff
@@ -1,18 +1,17 @@
...
 
 const pool = new Pool({
-    user: global.config.user,
-    host: global.config.host,
-    database: global.config.database,
-    password: global.config.password,
-    port: global.config.port,
+    user: process.env.DATABAE_USER,
+    host: process.env.DATABAE_HOST,
+    database: process.env.DATABAE,
```
Comment (velddev) [inline src/routes/users.router.ts]: Spelling issues

File: src/routes/users.router.ts [hunk 12]
```diff
@@ -1,18 +1,17 @@
...
+import axios from "axios";
 
 const pool = new Pool({
-    user: global.config.user,
-    host: global.config.host,
-    database: global.config.database,
-    password: global.config.password,
-    port: global.config.port,
+    user: process.env.DATABAE_USER,
+    host: process.env.DATABAE_HOST,
```
Comment (velddev) [inline src/routes/users.router.ts]: Spelling issues

File: src/routes/users.router.ts [hunk 13]
```diff
@@ -1,18 +1,17 @@
...
+import { parseString } from "xml2js";
+import axios from "axios";
 
 const pool = new Pool({
-    user: global.config.user,
-    host: global.config.host,
-    database: global.config.database,
-    password: global.config.password,
-    port: global.config.port,
+    user: process.env.DATABAE_USER,
```
Comment (velddev) [inline src/routes/users.router.ts]: Spelling issues

File: src/routes/users.router.ts [hunk 14]
```diff
@@ -1,18 +1,17 @@
...
+    host: process.env.DATABAE_HOST,
+    database: process.env.DATABAE,
+    password: process.env.DATABAE_PASSWORD,
+    port: Number(process.env.DATABAE_PORT)
```
Comment (velddev) [inline src/routes/users.router.ts]: Spelling issues

File: src/routes/users.router.ts (hunk 3 above)
Comment (velddev) [reply src/routes/users.router.ts]: the original style is `exp: number`

File: src/routes/users.router.ts [hunk 15]
```diff
@@ -52,16 +51,18 @@ function CalculateExp(level)
     return output;
 }
 
-router.get('/api/custom', async (req, res) => {
-    var xml = fs.readFileSync("./test.xml", "utf8");
+Router().get('/api/custom', async (req, res) => {
```
Comment (velddev) [inline src/routes/users.router.ts]: consider using "

File: src/routes/users.router.ts [hunk 16]
```diff
@@ -73,7 +74,7 @@ router.get('/api/custom', async (req, res) => {
     });
 });
 
-router.get('/api/ship', async (req, res) => {
+Router().get('/api/ship', async (req, res) => {
```
Comment (velddev) [inline src/routes/users.router.ts]: consider using "

File: src/routes/users.router.ts [hunk 17]
```diff
@@ -121,7 +123,7 @@ router.get('/api/ship', async (req, res) => {
     console.timeEnd('ship');
 });
 
-router.get('/api/user', async (req, res) =>
+Router().get('/api/user', async (req, res) =>
```
Comment (velddev) [inline src/routes/users.router.ts]: consider using "

**Review by velddev — CHANGES_REQUESTED** [review]
code LGTM, some variables need to be synced now with spelling stuff

File: src/routes/users.router.ts (hunk 10 above)
Comment (Niputi) [reply src/routes/users.router.ts]: ups

**Pull request merged**